includes/Functions.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Shows a configured message when a payment is successful.
+ * This message can be configured at the Wordpress dashboard.
+ * Also note that the message will be shown
+ * if the short code has been inserted in a page.
+ *
+ * @see \IDPay\CF7\Admin\Menu::admin_table()
+ *
+ * @param $message
+ * @param $track_id
+ * @param $order_id
+ *
+ * @return string
+ */
+function filled_message( $message, $track_id, $order_id ) {
+    return str_replace( [ "{track_id}", "{order_id}" ], [
+        $track_id,
+        $order_id,
+    ], $message );
+}
+
+/**
+ * @param $db
+ * @param $order_id
+ * @param $trans_id
+ * @param $track_id
+ * @param $status
+ * @param $message
+ * @return void
+ */
+function create_callback_response($db, $order_id, $trans_id, $track_id, $status, $message)
+{    $tableName = 'cf7_callbacks';
+    $row = $db->get_row( $db->prepare( "SELECT * FROM " . $db->prefix . $tableName ." WHERE id='%s'", $order_id ) );
+    if ( $row == NULL ) {
+        $row = [
+            'id' => $order_id,
+            'response' => json_encode([
+                'trans_id'=>$trans_id,
+                'track_id'=>$track_id,
+                'status'=>$status
+            ]),
```

Comment:
there is no need to save these ids in this table. this table is being used just for redirecting